src/utils/urls.js

```diff
@@ -0,0 +1,14 @@
+export const getLoginUrl = () => (
+  `${process.env.REACT_APP_EGGHEAD_BASE_URL}/users/jwt?return_to=${window.location.href}`
+)
+
+export const guideUrl = 'https://instructor.egghead.io/guide'
+
+export const chatUrl = 'https://eggheadio.slack.com/messages/egghead-instructors/'
```

Comment:
This is being used in places that are for instructors, so IMO egghead-instructors makes more sense. egghead-chat is for whatever and is noisy, egghead-instructors is for instructors to talk specifically about egghead instructing, which is what the instructor center is for. At least that is my view :)